### DISCUSSION TITLE
prisma query is not executing on tsx file in nextjs

### DISCUSSION THREAD

**Discussion by 89itworld** [prisma/prisma#14569] [Q&A]
### Bug description

I am trying to execute simple prisma query in .tsx file but it's giving the following error. 

Please find the attachment.

<img width="1217" alt="Screenshot 2022-07-29 at 2 37 00 PM" src="https://user-images.githubusercontent.com/3927785/181725682-dba044ff-b04c-4b33-b001-db14c4e7f283.png">

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by nurul3101** [comment]
Hey @89itworld 👋 
Are you trying to use Prisma in frontend/browser?
Could you share a sample on how are you using Prisma?

**Reply by vladi-stevanovic** [reply]
👋 Hello @89itworld I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
